exchanges/orderbook/tranches.go

```diff
@@ -88,7 +89,7 @@ updates:
 				copy((*ts)[y:], (*ts)[y+1:])
 				*ts = (*ts)[:len(*ts)-1]
 			} else {
-				*ts = append((*ts)[:y], (*ts)[y+1:]...)
+				*ts = slices.Delete(*ts, y, y+1)
 			}
```

Comment:
🚧  These are the same thing in disguise.
Can't make a suggestion for line 87, but this should just be:
```suggestion
*ts = slices.Clip(slices.Delete(*ts, y, y+1))
```

Reply:
Though, I think actually this whole section might be better as a single `*ts = slices.DeleteFunc`